tests/test_manylinux.py

```diff
@@ -225,6 +225,66 @@ def test_build_wheel_with_binary_executable(docker_container):
         python_id, ['python', '-c', 'from testpackage import runit; print(runit(1.5))']).strip()
     assert output.strip() == '2.25'
 
+@pytest.mark.parametrize('with_dependency', ['0', '1'])
+def test_build_wheel_with_highest_policy_and_image_dependencies(with_dependency, docker_container):
+    # Always try to repair the wheel targeting manylinux1 policy
+    #
+    # with_dependency == 0
+    #   The python module has no dependencies that should be grafted-in and
+    #   uses versioned symbols not available on policies pre-dating the policy matching the image being tested.
+    # with_dependency == 1
+    #   The python module itself does not use versioned symbols but has a dependency that should be grafted-in that
+    #   uses versioned symbols not available on policies pre-dating the policy matching the image being tested.
+    # Since manylinux1 has no pre-dating policy, it's the only image/policy that should be able to repair those wheels.
+
+    policy, manylinux_id, python_id, io_folder = docker_container
+
+    docker_exec(manylinux_id, ['bash', '-c', 'cd /auditwheel_src/tests/testdependencies && WITH_DEPENDENCY={} python setup.py -v build_ext -f bdist_wheel -d /io'.format(with_dependency)])
+
+    filenames = os.listdir(io_folder)
+    orig_wheel = filenames[0]
+    assert 'manylinux' not in orig_wheel
+
+    repair_command = (
+        'LD_LIBRARY_PATH=/auditwheel_src/tests/testdependencies auditwheel -v repair --plat manylinux1_x86_64 -w /io /io/{orig_wheel}'
+    ).format(policy=policy, orig_wheel=orig_wheel)
+
+    if policy != 'manylinux1':
+        # we shall fail on non manylinux1 images
+        with pytest.raises(CalledProcessError):
+            docker_exec(manylinux_id, ['bash', '-c', repair_command])
+    return
```

Comment:
I think the indentation is wrong here, so the rest of the test will never run.

Reply:
right, will correct